src/qt/guiutil.cpp

```diff
@@ -67,7 +68,7 @@ namespace GUIUtil {
 
 QString dateTimeStr(const QDateTime &date)
 {
-    return date.date().toString(Qt::SystemLocaleShortDate) + QString(" ") + date.toString("hh:mm");
+    return QLocale::system().toString(date.date(), QLocale::ShortFormat) + QString(" ") + date.toString("hh:mm");
```

Comment:
this is not a `ci` change, but a gui refactor change? Should probably be separate

Reply:
Done in https://github.com/bitcoin-core/gui/pull/137

Reply:
^ merged :)